Makefile

```diff
@@ -217,10 +217,10 @@ test-extended:
 #
 # Example:
 #   make run
-run: export OS_OUTPUT_BINPATH=$(shell bash -c 'source hack/common.sh; echo $${OS_OUTPUT_BINPATH}')
+run: export OS_OUTPUT_BINPATH=$(shell bash -c 'source hack/lib/build/constants.sh; echo $${OS_OUTPUT_BINPATH}')
 run: export PLATFORM=$(shell bash -c 'source hack/common.sh; os::build::host_platform')
 run: build
-	$(OS_OUTPUT_BINPATH)/$(PLATFORM)/openshift start
+	$(PWD)/$(OS_OUTPUT_BINPATH)/$(PLATFORM)/openshift start
```

Comment:
Please replace `$(PWD)` with `$(OS_ROOT)` which you can get like this:
`run: export OS_ROOT=$(shell bash -c 'source hack/lib/init.sh; echo $${OS_ROOT}')`